{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -44,14 +44,14 @@ jobs:
     action: gh:neuro-actions/remote_debug@master
     args:
```

Comment:
What if we expose the local port (2211) as a parameter with a default 2211 value, wdyt?

Reply:
Good idea. Updated corresponding `action.yaml`